Source/JavaScriptCore/dfg/DFGJITCode.cpp

```diff
@@ -64,11 +112,81 @@ JITData::JITData(VM& vm, CodeBlock* codeBlock, const JITCode& jitCode, ExitVecto
             at(i) = codeBlock->globalObject();
             break;
         }
-        default:
+        case LinkerIR::Type::HavingABadTimeWatchpointSet: {
+            auto* globalObject = codeBlock->globalObject();
+            auto& watchpoint = m_watchpoints[indexOfWatchpoints++];
+            failure |= !attemptToWatch(codeBlock, globalObject->havingABadTimeWatchpointSet(), watchpoint);
```

Comment:
I think it'll be clearer if you use a `success` bool instead of `failure` and express this as:
```
success &= attemptToWatch(codeBlock, globalObject->havingABadTimeWatchpointSet(), watchpoint);
```

Ditto for below.

Reply:
Changed.